### PR TITLE
Remove "none" plurals

### PR DESCRIPTION
**Pull request by Faalagorn**
#### Summary
None

#### Purpose of change
Don't make "none" plurals to appear in Transifex.

#### Describe the solution
Remove `"str_pl": "none"` from affected strings.

#### Describe alternatives you've considered
None.

#### Testing
None.

#### Additional context
Should fix the 6 issues by https://www.transifex.com/user/profile/Aloxaf/ on Transifex.

### CODE REVIEW

File: data/json/items/generic.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "id": "animal",
     "symbol": "?",
     "color": "white",
-    "name": { "str": "animal", "str_pl": "none" },
+    "name": { "str": "animal"},
```
Comment (ZhilkinSerg) [inline data/json/items/generic.json:8]:
```suggestion
    "name": { "str": "animal" },
```

File: data/json/items/generic.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "id": "fire",
     "symbol": "$",
     "color": "red",
-    "name": { "str": "nearby fire", "str_pl": "none" },
+    "name": { "str": "nearby fire" },
```
Comment (Qrox) [inline data/json/items/generic.json:22]:
```suggestion
    "name": { "str_sp": "nearby fire" },
```

File: data/json/items/generic.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
     "id": "muscle",
     "symbol": "?",
     "color": "white",
-    "name": { "str": "muscle", "str_pl": "none" },
+    "name": { "str": "muscle" },
```
Comment (Qrox) [inline data/json/items/generic.json:34]:
```suggestion
    "name": { "str_sp": "muscle" },
```

File: data/json/items/generic.json [hunk 4]
```diff
@@ -45,7 +45,7 @@
     "id": "wind",
     "symbol": "?",
     "color": "white",
-    "name": { "str": "wind", "str_pl": "none" },
+    "name": { "str": "wind" },
```
Comment (Qrox) [inline data/json/items/generic.json:48]:
```suggestion
    "name": { "str_sp": "wind" },
```

File: data/json/items/generic.json [hunk 5]
```diff
@@ -59,7 +59,7 @@
     "id": "sunlight",
     "symbol": "?",
     "color": "white",
-    "name": { "str": "sun light", "str_pl": "none" },
+    "name": { "str": "sun light" },
```
Comment (Qrox) [inline data/json/items/generic.json:62]:
```suggestion
    "name": { "str_sp": "sun light" },
```

File: data/json/items/generic.json [hunk 6]
```diff
@@ -87,7 +87,7 @@
     "id": "apparatus",
     "symbol": "$",
     "color": "red",
-    "name": { "str": "a smoking device and a source of flame", "str_pl": "none" },
+    "name": { "str": "a smoking device and a source of flame" },
```
Comment (Qrox) [inline data/json/items/generic.json:90]:
```suggestion
    "name": { "str_sp": "a smoking device and a source of flame" },
```

**Review by Qrox — COMMENTED** [review]
Use `str_sp` to avoid generating incorrect plural forms.

File: data/json/items/generic.json [hunk 7]
```diff
@@ -5,7 +5,7 @@
     "id": "animal",
     "symbol": "?",
     "color": "white",
-    "name": { "str": "animal", "str_pl": "none" },
+    "name": { "str": "animal" },
```
Comment (ZhilkinSerg) [inline data/json/items/generic.json:8]:
```suggestion
    "name": { "str_sp": "animal" },
```

**Pull request merged**